html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>HTML-form</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="#" method="POST">
+      <div>
+        <label for="email">Email:</label> <br />
+        <input type="email" name="txtEmail" id="email" />
+      </div>
+      &nbsp;
+
+      <div>
+        <label for="dropdown">Why is coding cool?</label><br />
+        <select name="cboDropdown" id="dropdown">
+          <option value="">Select an option</option>
+          <option value="">Because it's interesting</option>
+          <option value="">Because it's fun</option>
+          <option value="">Because it's powerful</option>
+        </select>
+      </div>
+      &nbsp;
+
+      <div>
+        <label for="radio">Favorite Language (choose one)</label><br />
```

Comment:
This should be a `p` element rather than a `label`. This `label` isn't connected to any form control elements